File: Kinvey Quickstart/TestObject.m
//
//  TestObject.m
//  Kinvey Quickstart
//
//  Created by Michael Katz on 11/12/12.
//  Copyright (c) 2012 Kinvey. All rights reserved.
//

#import "TestObject.h"

@implementation TestObject

- (NSDictionary *)hostToKinveyPropertyMapping
{
    return  @{ @"objectId" : KCSEntityKeyId, @"name" : @"name" };
}

@end

File: Kinvey Quickstart/TestObject.swift
//
//  TestObject.swift
//  Kinvey Test Drive
//
//  Created by Victor Hugo on 2017-02-27.
//  Copyright © 2017 Kinvey. All rights reserved.
//

import Kinvey

@objc
class TestObject: Entity {
    
    dynamic var name: String?
    
    override static func collectionName() -> String {
        return "testObjects"
    }
    
    override func propertyMapping(_ map: Map) {
        super.propertyMapping(map)
        
        name <- ("name", map["name"])
    }
    
}
